src/makers/win32/appx.js

```diff
@@ -65,7 +65,10 @@ export default async (dir, appName, targetArch, forgeConfig, packageJSON) => { /
     opts.devCert = await createDefaultCertificate(opts.publisher, outPath);
   }
 
-  if (!opts.publisher.match(/^CN=/)) {
+  if (typeof opts.publisher.match !== 'function') {
```

Comment:
You could, but it is not as direct. Alternatively you could just replace 
```
if (!opts.publisher.match(/^CN=/))
```
with
```
if (!opts.publisher || !opts.publisher.match(/^CN=/))
```
but I think that is less expressive.